lib/modules/hosts/superstack.js

```diff
@@ -0,0 +1,42 @@
+import { ajax } from '../../environment';
+
+const IMAGE = 'IMAGE';
+const GALLERY = 'GALLERY';
+
+const superstack = {
+	moduleID: 'superstack',
+	name: 'superstack',
+	domains: ['superstack.io'],
+	logo: '//superstack.io/favicon.ico',
+	detect({ href }) {
```

Comment:
This can be simplified to: `detect: ({ pathname }) => (/^\/v\/(\w+)(?:\/|$)/).exec(pathname),`